froide/account/forms.py

```diff
@@ -134,18 +136,41 @@ class NewUserBaseForm(AddressBaseForm):
         ),
     )
 
+    if USER_CAN_CLAIM_VIP:
+        claims_vip = forms.TypedChoiceField(
+            required=False,
+            initial=False,
+            widget=BootstrapRadioSelect,
+            label=_("{site_name} for journalists").format(site_name=settings.SITE_NAME),
+            help_text=_(
+                _(
+                    "You work in journalism and would like to use {site_name} for your research? Shortly after your sign-up is completed, we will send you additional information about extra functionality for journalists."
+                ).format(site_name=settings.SITE_NAME)
+            ),
+            choices=[
+                (False, _("No, I am <strong>not a journalist</strong>")),
+                (True, _("Yes, I am <strong>a journalist</strong>")),
+            ],
+            coerce=lambda x: x and (x.lower() != "false"),
+        )
+
     ALLOW_BLOCKED_ADDRESS = True
 
     if USER_CAN_HIDE_WEB:
-        private = forms.BooleanField(
+        private = forms.TypedChoiceField(
             required=False,
-            widget=BootstrapCheckboxInput,
+            widget=BootstrapRadioSelect,
             label=_("Hide my name from public view"),
-            help_text=mark_safe(
-                _(
-                    "If you check this, your name will still appear in requests to public bodies, but we will do our best to not display it publicly. However, we cannot guarantee your anonymity"
-                )
-            ),
+            choices=[
+                (
+                    False,
+                    _(
+                        "My name may appear on the website in <strong>plain text</strong>"
+                    ),
+                ),
+                (True, _("My name must be <strong>redacted</strong>")),
```

Comment:
something like "we will do our best to not display it publicly. However, we cannot guarantee your anonymity" should stay as a warning